src/binary-reader.cc

```diff
@@ -2280,6 +2333,12 @@ Result BinaryReader::ReadCustomSection(Index section_index,
     CHECK_RESULT(ReadTargetFeaturesSections(section_size));
   } else if (section_name == WABT_BINARY_SECTION_LINKING) {
     CHECK_RESULT(ReadLinkingSection(section_size));
+  } else if (options_.features.code_metadata_enabled() &&
+             section_name.find(WABT_BINARY_SECTION_CODE_METADATA) == 0) {
```

Comment:
Is there no `startswith` for C++ strings?

Reply:
it seems that they added it in c++20. But wabt uses c++17 currently.